lib/lightning_web/live/credential_live/credential_form_component.ex

```diff
@@ -688,16 +688,9 @@ defmodule LightningWeb.CredentialLive.CredentialFormComponent do
        ) do
     user_id = Ecto.Changeset.fetch_field!(changeset, :user_id)
 
-    project_credentials =
-      Ecto.Changeset.fetch_field!(changeset, :project_credentials)
-      |> Enum.map(fn %{project_id: project_id} ->
-        %{"project_id" => project_id}
-      end)
```

Comment:
This overwrites the associations that were set previously.

Reply:
Hey @jyeshe , did you try to edit it after creation? I think this was set so that you can edit as a whole

Reply:
Just tested. Works!!

Reply:
Thanks Frank, it was a surgical on `:new`